packages/react-admin/src/mui/modals/Modals.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import * as actions from './actions';
```

Comment:
Why import every possible actions ? That should be the responsibility of the user inside its render callback

Reply:
Same as the Modal comment. They are exported as redux actions to the render prop.